src/proteomes/config/ProteomesEntryConfig.tsx

```diff
@@ -0,0 +1,23 @@
+import Overview from '../components/entry/Overview';
+import { ProteomesUIModel } from '../adapters/proteomesConverter';
+import EntrySection from '../types/entrySection';
+
+const UniRefEntryConfig: {
```

Comment:
I just noticed this copy-paste relic and it will be fixed in a future PR (Components table).